main.js

```diff
@@ -0,0 +1,64 @@
+
+snakewaterMontana = {
+    uestOfHonor = ["Snakewater", "Montana"];
+      paleontologist: "Dr. Alan Grant",
+      depth: "10 meters",
+      specimen: "Velociraptor"
+    }
+  var guestOfHonor = 'paleontologist';
```

Comment:
in this part, we need to access the paleontologist value programmatically not manually so you need to go to the object first `snakewaterMontana` then the property of this object the solution will be something like this ` var guestOfHonor = snakewaterMontana.paleontologist;`